src/Services/UserService.ts

```diff
@@ -67,4 +66,23 @@ export class UserService {
         }
         return options
     }
+
+    public async updateAccount(userId: number, email: string, password: string) {
+        let updateParams: object;
+        if (password && email) {
```

Comment:
email and password are not optional arguments here and this should evaluate as true always?